### PR TITLE
[ENH] Fixed some PEP8 issues in example gallery.

### PR DESCRIPTION
**Pull request by emmanuelle**
Only very minor stuff.

### CODE REVIEW

File: doc/examples/plot_polygon.py [hunk 1]
```diff
@@ -18,26 +18,26 @@
...
                  [2.25403226, 2.60416667],
-                 [2.14919355, 1.953125  ],
+                 [2.14919355, 1.953125],
                  [2.30645161, 2.36979167],
                  [2.39112903, 2.36979167],
-                 [2.41532258, 2.1875    ],
-                 [2.1733871 , 1.703125  ],
+                 [2.41532258, 2.1875],
+                 [2.1733871, 1.703125],
                  [2.07782258, 1.16666667]])
```
Comment (jni) [inline doc/examples/plot_polygon.py]: @emmanuelle I would revert the changes to this file, because it is common and useful to align brackets and commas to array dimensions.

**Comment by jni** [comment]
@emmanuelle this is me cleaning up your mess! =P Only one very minor comment.

File: doc/examples/plot_polygon.py (hunk 1 above)
Comment (emmanuelle) [inline doc/examples/plot_polygon.py]: @jni OK, I understand your point but it's very annoying to have all these messages when doing ``pep8 *.py`` inside the gallery directory...

**Comment by blink1073** [comment]
I'm with @emmanuelle, merging.
